lib/absinthe/phase/document/variables.ex

```diff
@@ -76,4 +72,19 @@ defmodule Absinthe.Phase.Document.Variables do
     {node, acc}
   end
 
+  defp get(map, key, default) do
+    if Map.has_key?(map, key) do
+      Map.get(map, key) |> Blueprint.Input.parse
+    else
+      case {Map.get(map, key), default} do
```

Comment:
Could this be simplified by passing the default value directly to `Map.get`?

```elixir
case Map.get(map, key, default) do
  nil -> nil
  value -> Blueprint.Input.parse(value)
end
```

I think that covers all of the same cases.